docs/GettingStarted.md

```diff
@@ -147,7 +147,11 @@ Jest can be used in projects that use [webpack](https://webpack.js.org/) to mana
 
 Jest can be used in projects that use [parcel-bundler](https://parceljs.org/) to manage assets, styles, and compilation similar to webpack. Parcel requires zero configuration. Refer to the official [docs](https://parceljs.org/docs/) to get started.
 
-### Using TypeScript
+### Using TypeScript via ts-jest
+
+[ts-jest](https://github.com/kulshekhar/ts-jest) is a TypeScript preprocessor with source map support for Jest that lets you use Jest to test projects written in TypeScript.
```

Comment:
but can we mention the Babel approach first? That's the one that's more "native" in Jest. `ts-jest` as its own bullet right below seems sane 👍

Reply:
yeah sure, let me just push the change